function/webhook.go

```diff
@@ -86,13 +87,10 @@ func doWebhookReceived(creq CallRequest) error {
 			}
 		}
 
-		err = creq.StoreEvent(s.GoogleEmail, s.CalendarID, &Event{
+		_ = creq.StoreEvent(s.GoogleEmail, s.CalendarID, &Event{
 			Event:      e,
 			RootPostID: rootPostID,
 		})
-		if err != nil {
-			problems = append(problems, err)
-		}
```

Comment:
`problems` is already got out put in the bot DM. No way to properly output the err here